libs/deserializer/src/main/java/de/cyface/deserializer/v1/BinaryFormatDeserializer.java

```diff
@@ -0,0 +1,199 @@
+package de.cyface.deserializer.v1;
```

Comment:
I had to add copies of some classes to a separate v1 package so that we can use the old v1 models in those classes, to deserialized data in the file format v1.

This also ensures, we don't have to touch the `v1` code when we update our current v2 models as the v1 code uses it's own classes.

Reply:
I.e. this is an old copy of this class before we switched to the v2 model. I copied the classes from:

Collector v1 classes: [https://github.com/cyface-de/data-collector/tree/5.3.1/libs/model/src/main/java/de/cyface/model - Can't find link](https://github.com/cyface-de/data-collector/tree/5.3.1/libs/model/src/main/java/de/cyface/model) 

Backend v1 classes: [https://github.com/cyface-de/backend/tree/4.1.0/libs/surface/src/main/java/de/cyface/surface/operator - Can't find link](https://github.com/cyface-de/backend/tree/4.1.0/libs/surface/src/main/java/de/cyface/surface/operator)